src/app.js

```diff
@@ -15,7 +15,9 @@
         'openfin.star',
         'openfin.tearout',
         'openfin.minichart',
-        'openfin.scroll'
+        'openfin.scroll',
+        'openfin.closedwindows',
+        'openfin.closedcard'
```

Comment:
`minichart` is a bit of an oddball, perhaps it should be too. In any case, `currentWindow` and I think everything else should be.